File: Farhenit To Celcius/Farhenit.java
import java.util.*;

public class Farhenit{
    public static void main(String[] args) {
        Scanner in = new Scanner(System.in);
        float temperature,celcius;
        System.out.println("Enter temperature in farhenit:");
        temperature = in.nextFloat();
        celcius = ((temperature - 32) * 5) / 9;
        System.out.println("Temerature in celcius:" +celcius);
    }
}

File: Farhenit To Celcius/Farhenit.c
#include<stdio.h>
#include<stdlib.h>
#include<stdio.h>

int main()
{
    float temperature,celcius;
    printf("Enter tempereture in farhenit:");
    scanf("%f", &temperature);
    celcius = ((temperature - 32) * 5) / 9;
    printf("Temperature in Celcius = %f ",celcius);
    return 0;
}